eodatasets3/scripts/prepare.py

```diff
@@ -13,7 +13,7 @@ def run():
 
 
 run.add_command(landsat_l1, name="landsat-l1")
-run.add_command(sin_sentinel_l1, name="sin-sentinel-l1")
+run.add_command(sentinel_l1c, name="sentinel-l1C")
```

Comment:
```suggestion
run.add_command(sentinel_l1c, name="sentinel-l1")
```

Keep lowercase, like the others. And we probably don't need the `C`. 

... Landsat is just `landsat-l1`, so we my as well match it.